all_test.go

```diff
@@ -116,3 +120,57 @@ func rungo(t *testing.T, args ...string) {
 		t.Fatalf("%v: %v\n%s", cmd, err, output)
 	}
 }
+
+func TestExportedSymbolsHaveDocs(t *testing.T) {
+	// TODO(https://github.com/googleapis/librarian/issues/522): turn on once
+	// all existing symbols have docs.
+	t.Skip()
+
+	err := filepath.WalkDir(".", func(path string, d os.DirEntry, err error) error {
+		if err != nil || d.IsDir() || !strings.HasSuffix(path, ".go") || strings.HasSuffix(path, "_test.go") {
+			return nil
+		}
+
+		fset := token.NewFileSet()
+		node, err := parser.ParseFile(fset, path, nil, parser.ParseComments)
+		if err != nil {
+			t.Errorf("failed to parse file %q: %v", path, err)
+			return nil
+		}
+
+		// Visit every top-level declaration in the file.
+		for _, decl := range node.Decls {
+			gen, ok := decl.(*ast.GenDecl)
+			if ok && (gen.Tok == token.TYPE || gen.Tok == token.CONST || gen.Tok == token.VAR) {
+				for _, spec := range gen.Specs {
+					switch s := spec.(type) {
+					case *ast.TypeSpec:
+						checkDoc(t, s.Name, s.Doc, path)
+					case *ast.ValueSpec:
+						for _, name := range s.Names {
+							checkDoc(t, name, gen.Doc, path)
+						}
+					}
+				}
+			}
+			if fn, ok := decl.(*ast.FuncDecl); ok {
+				checkDoc(t, fn.Name, fn.Doc, path)
+			}
+		}
+		return nil
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func checkDoc(t *testing.T, name *ast.Ident, doc *ast.CommentGroup, path string) {
+	t.Helper()
+	if !name.IsExported() {
+		return
+	}
+	if doc == nil {
+		t.Errorf("%s: %q is missing doc comment",
```

Comment:
Just for my curiosity: is there a reason this is split across lines, but the error at line 137 isn't? (I thought line breaks were really uncommon in Go, but I may be wrong or there may be a different convention at work here.)

Reply:
Line breaks are at the author's discretion. gofmt will not insert them: there is no maximum line length. We add them when we feel the line is getting long. My usual editor window is about 120 chars wide, maybe more, and I will break when the line wraps on that. But there's no rule, and Julie's break here is consistent with common practice.